README.md

```diff
@@ -17,7 +17,8 @@ For the TLDR version:
 
 - The entry point is `ValidateOpenApiTask.ExecuteAsync()` and will be executed after the referencing project is built. This is defined in `./src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets` as a `UsingTask.TaskName`
 - The default value are defined in the property group on the target `ValidateOpenApi` in this file `./src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets`
-- Users can select whether to validate the API with frontend or backend ruleset depending how they configure the `OpenApiServiceProfile` MSBuild property ([`backend` (default)](https://github.com/gsoft-inc/wl-api-guidelines/blob/main/.spectral.backend.yaml) or [`frontend`](https://github.com/gsoft-inc/wl-api-guidelines/blob/main/.spectral.frontend.yaml)).
+- Users can select whether to validate the API with frontend or backend ruleset depending on how they configure the `OpenApiServiceProfile` MSBuild property ([`backend` (default)](https://github.com/gsoft-inc/wl-api-guidelines/blob/main/.spectral.backend.yaml) or [`frontend`](https://github.com/gsoft-inc/wl-api-guidelines/blob/main/.spectral.frontend.yaml)).
+- Users can configure their CI enviroment for reports depending on how they configure the `OpenApiCiReportEnvironment` MSBuild property (`ado` (default)).
```

Comment:
Do we need the developer to provide this information? We can detect whether or not the build is running on ADO using well-known environment variables, like we already do here:

https://github.com/gsoft-inc/wl-extensions-xunit/blob/bd7d883917a1915b5af54d4fe848594bc4dab1ca/src/Workleap.Extensions.Xunit/BaseIntegrationFixture.cs#L77-L79

Reply:
Also I don't understand why we sometimes use this user-provided information, then sometimes look up for an environment variable

Reply:
Discussed offline and full agree with that, we'll default to using environment variables present on the CI build env